custom_md_loader/pages/CMTMigrator.page

```diff
@@ -6,93 +6,98 @@
  */
 -->
 <apex:page controller="MetadataMigrationController" docType="html-5.0" sidebar="false">
+	<style type="text/css">
+    	.pbTitle {
+	        white-space: nowrap;
+    	}
+	</style>
 
-	<body class="homeTab">
-
+	<body>
       <apex:form id="form">
-
        <apex:pageBlock title="Messages" rendered="{!isMessage == true}">
             <apex:pageMessages />
         </apex:pageBlock>
 
 		<apex:pageBlock title="Custom Settings/Custom Objects to Custom Metadata Types Migration">
 			<ol>
```

Comment:
It's kind of weird to have this ordered list in the user interface. I think it would be a little easier to understand if you put the descriptions of the four options in the same pageblocks as the options themselves, and don't number the rest of the information.